### ISSUE TITLE
Wire damage detector + ACF Engine Infinite Loop

### ISSUE THREAD

**Issue by thegrb93** [wiremod/wire#904]
I downloaded a dupe off of some idiot who was using it to crash my server. This is the result.
![crash](http://i.imgur.com/yHkTzRe.png)

Heres a table describing the dupe. http://pastebin.com/2DK686eF
It is using these entities:

gmod_wire_damage_detector (linked to itself) (wired to the constant value)
acf_engine (wired to the damage detector)
gmod_wire_value
prop_physics models/props_phx/mk-82.mdl

Basically once the mk-82 pops, this crash occurs.

I think it is because once the detector is activated, the engine does damage on the detector, which then revs the engine, which then does damage to the detector, etc. Not sure though.

**Comment by thegrb93** [comment]
Haven't seen this again and the dupe I used to crash the game doesn't work anymore so I guess an update fixed it.
